modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayCases.macro

```diff
@@ -1,5 +1,19 @@
 definition {
 
+	macro linkCasesRequirement {
+		TestrayRequirements.addRequirementCase(
+			caseName = "${requirementName}",
+			linkText = "${linkText}",
+			buttonText = "${buttonText}");
+	}
+
+	macro verifyCaseRequirement {
+		TestrayRequirements.verifyRequirementCase(
+			requirementCaseName = "ATF_Requirements Case",
+			requirementComponent = "Components_1 Project 1",
+			requirementPriority = "5");
+	}		
```

Comment:
Hey @joelrocha68, I think that you can put variables on this macro, instead of the defined values